docs/user-guide/layer-settings.md

```diff
@@ -341,6 +353,22 @@ The style editor can configure the *Icon* as explained [here](#icon) along with
 
 <img src="../img/layer-settings/classify_icon_ex.jpg" class="ms-docimage">
 
+#### Style with properties
+
+[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows to choose a property of a layer's feature as the value for a symbolizer option. The *Style with properties* function is available for *Marker*, *Icon*, *Line*, *Fill*, *Text* and *3D model* by clicking on the <img src="../img/button/options_button.jpg" class="ms-docbutton"/> button of each symbolizer option supported by this function and choosing **Property value** option from the dropdown menu.
+
+<img src="../img/layer-settings/properties_style.jpg" class="ms-docimage"  style="max-width:500px;">
+
+By default, **Constant value** is selected. If the user chooses **Property value**, a drop-down menu appears so that the desired layer attribute can be selected to use the necessary values from the styling property.
```

Comment:
```suggestion
By default, **Constant value** is selected. If the user chooses **Property value**, a drop-down menu appears so that the desired layer attribute can be selected to use its values for the styling property.
```